docs/docs/04_docker_intro/03_run_docker_container/README.md

```diff
@@ -0,0 +1,123 @@
+# How to run a Docker container
+
+## Install Docker Desktop
+
+Docker Desktop is an application to help you manage your images and containers. Download it and install it here: [https://www.docker.com/products/docker-desktop/](https://www.docker.com/products/docker-desktop/).
+
+## Create your Docker image
+
+Next, download the REST API code from Section 3. You can download it here: LINK.
```

Comment:
I guess that in the future the missing LINK will be created.

Reply:
Yes! I will do this before release.